runtime/runtime.go

```diff
@@ -2961,7 +2981,8 @@ func (r *interpreterRuntime) newAccountContractsGetNamesFunction(
 
 		values := make([]interpreter.Value, len(names))
 		for i, name := range names {
-			values[i] = interpreter.NewStringValue(name)
+			// TODO: meter?
+			values[i] = interpreter.NewUnmeteredStringValue(name)
```

Comment:
I think we should meter all cadence-values that get created at the runtime interface (e.g: values passed over from FVM), since its one of the three ways a value can be constructed in cadence (i.e: in-line, program-arguments, sent from FVM)

Reply:
Agreed, I will address this in a follow-up PR 👍